src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsAutoListMapping.java

```diff
@@ -67,6 +68,10 @@ public static boolean isThereOtherComments(@Valid SscsCaseData caseData) {
         return isNotBlank(HearingsDetailsMapping.getListingComments(caseData));
     }
 
+    public static boolean doesNotHaveDwpResponseDate(@Valid SscsCaseData caseData) {
+        return !isNotBlank(caseData.getDwpResponseDate());
```

Comment:
Maybe we can use `isBlank(....)` instead of !isNotBlank for readability(just a suggestion)